xknx/devices/remote_value_dpt3.py

```diff
@@ -0,0 +1,215 @@
+
+        ret = DPTBinary(0)
+        if abs(value) >= 100:
+            ret = DPTBinary(sign << 3 | 1)
+        elif abs(value) >= 50:
+            ret = DPTBinary(sign << 3 | 2)
+        elif abs(value) >= 25:
+            ret = DPTBinary(sign << 3 | 3)
+        elif abs(value) >= 12:
+            ret = DPTBinary(sign << 3 | 4)
+        elif abs(value) >= 6:
+            ret = DPTBinary(sign << 3 | 5)
+        elif abs(value) >= 3:
+            ret = DPTBinary(sign << 3 | 6)
+        elif abs(value) >= 1:
+            ret = DPTBinary(sign << 3 | 7)
+        return ret
+
+    def from_knx(self, payload):
+        """Convert current payload to value."""
+        if payload.value & ~0x0F != 0:  # more than 4-bit
+            raise CouldNotParseTelegram("payload invalid", payload=payload, device_name=self.device_name)
+        # calculated using floor(100/2^((value&0x07)-1))
+        value = [0, -100, -50, -25, -12, -6, -3, -1, 0, 100, 50, 25, 12, 6, 3, 1][payload.value & 0x0F]
+        return value if not self.invert else -value
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit of measurement."""
+        return "%"
+
+
+class RemoteValueStartStopDimming(RemoteValue):
+    """Abstraction for remote value of KNX DPT 3.007 / DPT_Control_Dimming."""
+
+    class Direction(Enum):
+        """Enum for indicating the direction."""
+
+        DECREASE = 0
+        INCREASE = 1
+        STOP = 2
+
+        def __str__(self):
+            """Return string representation."""
+            return self.name
+
+    def __init__(self,
+                 xknx,
+                 group_address=None,
+                 group_address_state=None,
+                 device_name=None,
+                 after_update_cb=None,
+                 invert=False):
+        """Initialize remote value of KNX DPT 3.007."""
+        # pylint: disable=too-many-arguments
+        super(RemoteValueStartStopDimming, self).__init__(
+            xknx, group_address, group_address_state,
+            device_name=device_name, after_update_cb=after_update_cb)
+        self.invert = invert
+
+    def payload_valid(self, payload):
+        """Test if telegram payload may be parsed."""
+        return isinstance(payload, DPTBinary)
+
+    def to_knx(self, value):
+        """Convert value to payload."""
+        if value == self.Direction.INCREASE:
+            return DPTBinary(0x09) if not self.invert else DPTBinary(0x01)
+        if value == self.Direction.DECREASE:
+            return DPTBinary(0x01) if not self.invert else DPTBinary(0x09)
+        if value == self.Direction.STOP:
+            return DPTBinary(0x00)
+        raise ConversionError("value invalid", value=value, device_name=self.device_name)
+
+    def from_knx(self, payload):
+        """Convert current payload to value."""
+        if payload.value & ~0x0F != 0:  # more than 4-bit
+            pass  # raises exception below
+        elif payload.value & 0x07 == 0:
```

Comment:
Just that i understand this right, this evaluates for `value=8` and `value=0` ...?

(I'm not too good in bitwise calculations, bit i think we should be more explicit here:

`if payload.value in [0, 8]:` if this is intended.